Infrastructure/Componants/TopMenu.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using Core;
+namespace Infrastructure
+{
+    public class TopMenu : ComponantBase
+    {
+        private IWebElement WomenButton => ParentElement.FindElements(By.CssSelector("li a")).First();
```

Comment:
What if the list changes somehow? Maybe you can get the element by the title='Women' attribute
You can google how to get css selector by attribute